contracts/contracts/svg/partials/stone.hbs

```diff
@@ -1,32 +1,28 @@
 <filter id="texture">
   <feTurbulence
-    type={{turbType}}
-    id="octave"
-    baseFrequency="{{turbFreqX}} {{turbFreqY}}"
-    numOctaves="{{turbOct}}"
-    seed="1004123123"
+    {{#if fractalNoise}}type="fractalNoise"{{/if}}
+    baseFrequency="{{uint8 turbFreqX decimals=3}} {{uint8 turbFreqY decimals=3}}"
+    numOctaves="{{uint8 turbOct}}"
+    seed="{{uint seed}}"
   ></feTurbulence>
   <feComponentTransfer>
     <feFuncR
-      id="r"
       type="gamma"
-      amplitude="{{redAmp}}"
-      exponent="{{redExp}}"
-      offset="{{redOff}}"
+      amplitude="{{int8 redAmp decimals=2}}"
```

Comment:
nice, i'm assuming storing ints is less space than floats?

Reply:
Solidity doesn't have floats, so ints with fixed points and are our only option